spark/src/main/scala/geotrellis/spark/Implicits.scala

```diff
@@ -267,4 +268,11 @@ trait Implicits
       TileLayerMetadata.fromRDD[K1, V, K2](rdd, layout)
     }
   }
+
+  implicit class withGetBoundsMethod[K: Boundable, V <: CellGrid[Int]](rdd: RDD[(K, V)]) {
```

Comment:
Wondering is it used anywhere?

Reply:
I don't think it is. That method used to be [on KeyBounds](https://github.com/locationtech/geotrellis/blob/master/spark/src/main/scala/geotrellis/spark/KeyBounds.scala#L95-L98), so I just moved it in order to retain the functionality.